sdks/universal-router-sdk/README.md

```diff
@@ -31,3 +80,6 @@ Run forge integration tests
 forge install
 yarn test:forge
 ```
+
+## Contributing
+Contributions to improve the SDK or documentation are welcome. Please ensure you follow the existing code style and add appropriate tests for any new functionality.
```

Comment:
remove, we can create a separate CONTRIBUTING.md in the future

Reply:
removed.